app/renderer/js/pages/preference/general-section.ts

```diff
@@ -389,51 +394,51 @@ class GeneralSection extends BaseSection {
 		});
 	}
 
-	addCustomCSS() {
+	addCustomCSS(): void {
 		const customCSSButton = document.querySelector('#add-custom-css .custom-css-button');
 		customCSSButton.addEventListener('click', () => {
 			this.customCssDialog();
 		});
 	}
 
-	showCustomCSSPath() {
+	showCustomCSSPath(): void {
 		if (!ConfigUtil.getConfigItem('customCSS')) {
-			const cssPATH = document.getElementById('remove-custom-css');
-			cssPATH.style.display = 'none';
+			const cssPATH = document.querySelector('#remove-custom-css');
+			(cssPATH as HTMLElement).style.display = 'none';
 		}
 	}
 
-	removeCustomCSS() {
-		const removeCSSButton = document.getElementById('css-delete-action');
+	removeCustomCSS(): void {
+		const removeCSSButton = document.querySelector('#css-delete-action');
 		removeCSSButton.addEventListener('click', () => {
-			ConfigUtil.setConfigItem('customCSS');
+			ConfigUtil.setConfigItem('customCSS', undefined);
```

Comment:
Let's make this `""`, a blank string, since we later pull out this value an inject it into the style tag anyways. cc @akashnimare